src/corehost/cli/fxr_resolver.h

```diff
@@ -46,22 +46,37 @@ int load_fxr_and_get_delegate(hostfxr_delegate_type type, THostNameToAppNameCall
 
     // Leak fxr
 
-    auto get_delegate_from_hostfxr = (hostfxr_get_delegate_fn)pal::get_symbol(fxr, "hostfxr_get_runtime_delegate");
-    if (get_delegate_from_hostfxr == nullptr)
+    auto hostfxr_init_context = (hostfxr_initialize_for_runtime_config_fn)pal::get_symbol(fxr, "hostfxr_initialize_for_runtime_config");
```

Comment:
Nit: can we use the real function name - I was looking for a helper called `hostfxr_init_context` since it's not the name of the actual export.